README.md

```diff
@@ -1,10 +1,106 @@
-OpenCensus - A stats collection and distributed tracing framework
-=================================================================
+# OpenCensus - A stats collection and distributed tracing framework
 [![Build Status][travis-image]][travis-url] [![Build status][appveyor-image]][appveyor-url] [![Maven Central][maven-image]][maven-url]
 
-OpenCensus provides a framework to define and collect stats against metrics and to
-break those stats down across user-defined dimensions. The library is in alpha
-stage and the API is subject to change.
+OpenCensus provides a framework to define and collect stats against metrics and to break those 
+stats down across user-defined dimensions. The library is in alpha stage and the API is subject 
+to change.
+
+## Quickstart for Third Party Libraries
+
+In general third party libraries only need to record tracing/stats events and propagate the context.
```

Comment:
Integrating census with a new library starts means recording stats or traces and propagating context.

Reply:
done.